### DISCUSSION TITLE
Add RTSP video stream support

### DISCUSSION THREAD

**Discussion by patrickelectric** [godotengine/godot-proposals#6128] [Engine Core]
### Describe the project you are working on

Create a game like Mario kart home edition
![mario_kart_live_home_circuit_839_10_becb84353bddfae3d4c9e806eede64ee_20220224082118](https://user-images.githubusercontent.com/1215497/213586843-688b8c2e-2dee-4a68-b729-fd8315e28059.jpg)

### Describe the problem or limitation you are having in your project

There is no real time video source support 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Adding support to RTSP video stream 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

No idea, but it would be nice to have "rtsp://IP:port/video0" as a valid source for the video stream 

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Nops

### Is there a reason why this should be core and not an add-on in the asset library?

Not sure

**Comment by Calinou** [comment]
Moved to discussions, as there is no concrete technical implementation provided.

Related to https://github.com/godotengine/godot-proposals/issues/4710 (which is about making Godot able to act as a RTSP *server*, not client).

**Comment by patrickelectric** [comment]
There is this awesome plugin by well known @williangalvani
https://github.com/Williangalvani/udp-rtsp-gdextension
